django/publicmapping/publicmapping/urls.py

```diff
@@ -32,6 +32,7 @@
 
 urlpatterns = ([
     url(r'^$', publicmapping_views.index),
+    url(r'^health-check/$', include('watchman.urls')),
```

Comment:
I noticed a warning pass by for this line:

`Your URL pattern '^health-check/$' uses include with a regex ending with a '$'. Remove the dollar from the regex to avoid problems including URLs.`